Start using 'previous-tag' property

shestee opened the pull request:
Recently new property _'shipkit-auto-version.previous-tag'_ has been exposed. It allows for getting previous revision in more convenient way. This is done in release.gradle file with first commit of this PR - instead of _'previous-version'_ property, the _'previous-tag'_ is used. New property is now also covered with README.md documentation (accordig to @mockitoguy suggestions, corrected a bit to make it more readable).

shestee committed:
Started using 'previous-tag' property

Recently new property 'shipkit-auto-version.previous-tag' has been exposed. It allows for getting previous revision in more convenient way, what is hereby done in release.gradle file, instead of
using 'previous-version' property.
New property is now also covered with README.md documentation.

shestee committed:
Corrected properties section in documentation

To make previously added coverage on new property 'previous-tag' in README.md better organized and more readable, made some correction to this documentation section.

mockitoguy commented on README.md:
````diff
@@ -50,14 +50,27 @@ has negligible performance implication (adds ~2 secs to the checkout).
+```
+tasks.named("generateChangelog") {
+   previousRevision = project.ext.'shipkit-auto-version.previous-tag'
+   ...
````
... -> //...

(making the code working, will look cleaner with syntax highlighting)

mockitoguy commented on README.md:
````diff
@@ -50,14 +50,27 @@ has negligible performance implication (adds ~2 secs to the checkout).
+plugin as in example below).
+Example:
+
+```
````
\``` -> \```Gradle

(provides syntax highlighting in the code sample)

shestee replied:
Thanks! Just added the tweak

mockitoguy reviewed (changes requested):
Nice work! Requesting tweak to the documentation. Thanks!

shestee committed:
Improved documentation

In order to make properties usage samples clearer and easier to read they're highlighted.

mockitoguy reviewed (approved):
Thanks!
